pan-domain-node/test/panda.test.ts

```diff
@@ -13,28 +13,29 @@ import {decodeBase64} from "../src/utils";
 describe('verifyUser', function () {
 
     test("return invalid cookie if missing", () => {
-        expect(verifyUser(undefined, "", 0, guardianValidation).status).toBe(AuthenticationStatus.INVALID_COOKIE);
+        expect(verifyUser(undefined, "", new Date(0), guardianValidation).status).toBe(AuthenticationStatus.INVALID_COOKIE);
     });
 
     test("return invalid cookie for a malformed signature", () => {
         const [data, signature] = sampleCookie.split(".");
         const testCookie = data + ".1234";
 
-        expect(verifyUser(testCookie, publicKey, 0, guardianValidation).status).toBe(AuthenticationStatus.INVALID_COOKIE);
+        expect(verifyUser(testCookie, publicKey, new Date(0), guardianValidation).status).toBe(AuthenticationStatus.INVALID_COOKIE);
     });
 
     test("return expired", () => {
-        const someTimeInTheFuture = 5678;
+        const someTimeInTheFuture = new Date(5678);
+        expect(someTimeInTheFuture.getTime()).toBe(5678);
```

Comment:
Why did this test not catch the problem originally? Are there any extra tests or comments that could be added to help avoid or clarify that pitfall?

Reply:
Oh! It's because it doesn't actually test the `verify` function which is what's called by the consumer of this library 😂🤦🏻‍♂️

Let's **definitely** add a tests for the actual public API of the library:
- `stop()`
- `getPublicKey()`
- `verify()`

Reply:
several hours of fighting with jest's mocking mechanisms later, I've got a handful of tests added now

Reply:
You're an absolute hero! 🫡 Yeah it occurred to me after writing this comment that the annoyance of spoofing `new Date()` was probably what caused someone to avoid writing tests for the public API in the first place!